app/pb_kits/playbook/pb_filter/docs/_filter_default.html.erb

```diff
@@ -28,8 +28,12 @@
     ]
   %>
   <%= pb_rails("form", props: { form_system_options: { scope: :example, method: :get } }) do |form| %>
-    <%= form.text_field :example_text_field, props: { label: true } %>
-    <%= form.collection_select :example_collection_select, example_collection, :value, :name, props: { label: true } %>
+    <%# <%= form.text_field :example_text_field, props: { label: true } %>
+    <%# <%= form.collection_select :example_collection_select, example_collection, :value, :name, props: { label: true } %>
```

Comment:
Why comment these out? Are you just trying to stay focused on your testing?

Reply:
Wanted try using it inside the filter (BOP's use case), see how they oriented themselves and what the spacing looked like.  I'll discard these changes once and clean up the docs once it comes time to merge this into Playbook.